src/components/TrySection/index.js

```diff
@@ -9,14 +9,14 @@ export default function TrySection({ title, buttonText = 'Get Started' }) {
   return (
     <Wrapper>
       <Container>
-        <Glyph src="/static/img/glyph-3.svg" gid={'topleft'} />
+        <Glyph src="/img/glyph-3.svg" gid={'topleft'} />
         <Title>{title}</Title>
         <Buttons>
-          <LocalLink href="/doc/get-started" as={Button} first>
+          <LocalLink href="/doc/get-started" as={Button} first="true">
```

Comment:
Could you make sure this ends up being rendered as an `<a>`? You probably did, but just making sure for SEO/A11y purposes.

Reply:
I probably should rename this component, all `Button`s on this page are actually `styled.a`, but yes, the all are rendered to `a`.